src/main/java/com/nttdata/nttdatanotificationservice/rocket/models/RocketMessage.java

```diff
@@ -1,17 +1,23 @@
 package com.nttdata.nttdatanotificationservice.rocket.models;
 
 public class RocketMessage {
+
+    private static final String AVATAR = "https://user-images.githubusercontent.com/51387119/82707419-ddb1c600-9c30-11ea-8bfa-b3c624b23cdd.png";
+
     private String alias;
     private String avatar;
     private String emoji;
     private String text;
 
+    public RocketMessage(String alias, String avatar) {
+        this.alias = alias;
+        this.avatar = avatar;
+    }
+
     public String getAlias() { return alias; }
 
     public void setAlias(String alias) {  this.alias = alias; }
```

Comment:
setters can be removed, so your object becomes immutable ;-)

Reply:
Removed